src/app/pages/HomePage/components/Contact/Contact.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../../../../variables';
+.joininterno {
+    width: 100%;
+    height: 374px;
+    background: $text-color;
+    border-radius: 70px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 32px;
+    position: absolute;
```

Comment:
you don't need position: absolute for this block
also, you don't need to set the height: iy must be calculated from your content and your paddings